### PR TITLE
dev_env reorganized, indexed

### PR DESCRIPTION
**Pull request by kathpanderson**
Same things as in the past pull request put onto the newer files.

### CODE REVIEW

File: development/contributing/contributing-code.rst [hunk 1]
```diff
@@ -1,7 +1,12 @@
+.. index::
+   pair: Contributing; Code
+
+.. _contrib_code:
+
 Contributing to Digital Rebar Core
 ----------------------------------
 
-Before `submitting pull requests <https://help.github.com/articles/using-pull-requests>`_, please make sure to have read and understood the Apache license. Submitting a pull is considered to be accepting the project's license terms.
+Before `submitting pull requests <https://help.github.com/articles/using-pull-requests>`_, please make sure to have read and understand the Apache license. Submitting a pull is considered to be accepting the project's license terms.
```
Comment (galthaus) [inline development/contributing/contributing-code.rst]: Seems like we should get read of the have in "to have read"

**Comment by kathpanderson** [comment]
Fixed "to have read" in #118

**Comment by galthaus** [comment]
conflicts again

**Comment by kathpanderson** [comment]
Remade changes is pull #126.

**Pull request closed without merging**